balboa-http/src/main/scala/com/socrata/balboa/server/MainServlet.scala

```diff
@@ -37,9 +43,32 @@ class MainServlet extends ScalatraServlet
     }
   }
 
-  get("/entities*") {
-    val response = EntitiesRest(params)
-    contentType = response.contentType
-    response.result
+  get("/entities*")(getEntities)
+  def getEntities: ActionResult = {
+    contentType = json
+
+    val predicate: (String => Boolean) =
+      params.get("filter").map(Pattern.compile)
+        .map { pattern => (str: String) => pattern.matcher(str).matches }.getOrElse(_ => true)
+
+    timer("entities queries")({
```

Comment:
This is actually the ~forbidden~ API for fetching a list of all entities that have existing metrics. Results look like the following:

```
[
  "testMetricsEntity-603eecde-1769-4463-b585-1ce1f84be5bd",
  "testMetricsEntity-55f7ca73-fcf3-4fe4-ba44-62ead5db8219",
  "testMetricsEntity-2f605cd2-3fe1-4203-b11a-951a9ce0b49b"
]
```
So, it is actually getting entities, or at least their names.